### PR TITLE
Add me.alexdevs.Capy64

### PR DESCRIPTION
**Pull request by Ale32bit**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by Ale32bit** [comment]
bot, build me.alexdevs.Capy64

File: flathub.json
Comment (hfiguiere) [inline flathub.json:1]: remove this file.

File: me.alexdevs.Capy64.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+   ],
+   "command":"start-capy64",
+   "finish-args":[
+      "--socket=x11",
+      "--socket=wayland",
+      "--share=network",
+      "--share=ipc",
+      "--socket=pulseaudio",
+      "--device=dri",
+      "--filesystem=xdg-data/Capy64",
```
Comment (hfiguiere) [inline me.alexdevs.Capy64.json:17]: line 17:  why do you need this?

File: me.alexdevs.Capy64.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+   ],
+   "modules":[
+      {
+         "name":"capy64",
+         "buildsystem":"simple",
+         "build-options":{
+            "append-path":"/usr/lib/sdk/dotnet7/bin",
+            "append-ld-library-path":"/usr/lib/sdk/dotnet7/lib",
+            "env":{
+               "PKG_CONFIG_PATH":"/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet7/lib/pkgconfig"
```
Comment (hfiguiere) [inline me.alexdevs.Capy64.json:28]: use `append-pkg-config-path` instead.

File: msources.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    {
+        "type": "file",
+        "url": "https://globalcdn.nuget.org/packages/microsoft.aspnetcore.app.ref.7.0.9.nupkg",
+        "sha512": "0ad039da094355bfdb221d349d58279ee3bbfe890bf16731a0ff238fbd298f50e47b62b68cfa4e7d2bc3e47faa3e6849941f1060a9cda2ea5c7475146383c130",
+        "dest": "nuget-sources",
+        "dest-filename": "microsoft.aspnetcore.app.ref.7.0.9.nupkg"
+    },
+    {
+        "type": "file",
+        "url": "https://globalcdn.nuget.org/packages/microsoft.netcore.app.host.linux-x64.7.0.9.nupkg",
```
Comment (hfiguiere) [inline msources.json:18]: how will this work on aarch64?

**Review by hfiguiere — CHANGES_REQUESTED** [review]
desktop file, icon and appdata files should be submitted for inclusion upstream.

File: appdata.xml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  <description>
+    <p>
+      Capy64 is a fantasy console that runs sandbox Lua 5.4.
+    </p>
+    <p>
+      Create anything, from Hello Worlds to games!
+    </p>
+  </description>
+  
+  <launchable type="desktop-id">me.alexdevs.Capy64.desktop</launchable>
```
Comment (hfiguiere) [inline appdata.xml:26]: missing required `content_rating`, `url`, `releases` and `screenshots`

File: me.alexdevs.Capy64.json [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+   "runtime":"org.freedesktop.Platform",
+   "runtime-version":"22.08",
+   "sdk":"org.freedesktop.Sdk",
+   "sdk-extensions":[
+      "org.freedesktop.Sdk.Extension.dotnet7"
+   ],
+   "command":"start-capy64",
+   "finish-args":[
+      "--socket=x11",
+      "--socket=wayland",
```
Comment (hfiguiere) [inline me.alexdevs.Capy64.json:12]: does it really support wayland?

File: Capy64.desktop [hunk 6]
```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+
+Name=Capy64
+Comment=Lua Fantasy Computer
+Categories=Game;Emulator;
+
+Icon=me.alexdevs.Capy64
+Exec=
```
Comment (hfiguiere) [inline Capy64.desktop:10]: missing exec

**Pull request closed without merging**